code/game/gamemodes/cult/ritual.dm

```diff
@@ -1,698 +1,300 @@
+
+	text = jointext(text, "In addition to these runes, the cult has a small selection of equipment and constructs.<br><br>")
+
+	text = jointext(text, "<font color='red'><b>Equipment:</b></font><br><br>")
+
+	text = jointext(text, "<font color='red'><b>Cult Blade</b></font><br>Cult blades are sharp weapons that, notably, cannot be used by noncultists. These blades are produced by the Talisman of Arming.<br><br>")
+
+	text = jointext(text, "<font color='red'><b>Cult Bola</b></font><br>Cult bolas are strong bolas, useful for snaring targets. These bolas are produced by the Talisman of Arming.<br><br>")
+
+	text = jointext(text, "<font color='red'><b>Cult Robes</b></font><br>Cult robes are heavily armored robes. These robes are produced by the Talisman of Arming.<br><br>")
+
+	text = jointext(text, "<font color='red'><b>Soulstone</b></font><br>A soulstone is a simple piece of magic, produced either via the starter talisman or by sacrificing humans. Using it on an unconscious or dead human, or on a Shade, will trap their soul in the stone, allowing its use in construct shells. \
+	<br>The soul within can also be released as a Shade by using it in-hand.<br><br>")
+
+	text = jointext(text, "<font color='red'><b>Construct Shell</b></font><br>A construct shell is useless on its own, but placing a filled soulstone within it allows you to produce your choice of a <b>Wraith</b>, a <b>Juggernaut</b>, or an <b>Artificer</b>. \
+	<br>Each construct has uses, detailed below in Constructs. Construct shells can be produced via the starter talisman or the Rite of Fabrication.<br><br>")
+
+	text = jointext(text, "<font color='red'><b>Constructs:</b></font><br><br>")
+
+	text = jointext(text, "<font color='red'><b>Shade</b></font><br>While technically not a construct, the Shade is produced when released from a soulstone. It is quite fragile and has weak melee attacks, but is fully healed when recaptured by a soulstone.<br><br>")
+
+	text = jointext(text, "<font color='red'><b>Wraith</b></font><br>The Wraith is a fast, lethal melee attacker which can jaunt through walls. However, it is only slightly more durable than a shade.<br><br>")
+
+	text = jointext(text, "<font color='red'><b>Juggernaut</b></font><br>The Juggernaut is a slow, but durable, melee attacker which can produce temporary forcewalls. It will also reflect most lethal energy weapons.<br><br>")
+
+	text = jointext(text,"<font color='red'><b>Artificer</b></font><br>The Artificer is a weak and fragile construct, able to heal other constructs, produce more <font color='red'><b>soulstones</b></font> and <font color='red'><b>construct shells</b></font>, \
+	construct fortifying cult walls and flooring, and finally, it can release a few indiscriminate stunning missiles.<br><br>")
+
+	text = jointext(text, "<font color='red'><b>Harvester</b></font><br>If you see one, know that you have done all you can and your life is void.<br><br>")

```

Comment:
Ah, this can all be `+=`, not `jointext` - at the end, you can do `var/text_string = jointext(text, null)` - and then, use `text_string` instead of `text` inside `popup.set_content`